apps/console/src/features/identity-providers/api/identity-provider.ts

```diff
@@ -47,7 +47,8 @@ import {
     MultiFactorAuthenticatorInterface,
     OutboundProvisioningConnectorInterface,
     OutboundProvisioningConnectorListItemInterface,
-    OutboundProvisioningConnectorMetaInterface
+    OutboundProvisioningConnectorMetaInterface,
+    NotificationSenderSMSInterface
```

Comment:
Shall we fix this lint issue?

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a